cli/src/main/resources/config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In Spring Boot it shouldn't be necessary to have xml-files? Can't this be added as an annotation?